Patches/ExilePatch.cs

```diff
@@ -1,11 +1,10 @@
+using System.Linq;
 using AmongUs.Data;
 using HarmonyLib;
-using System.Linq;
```

Comment:
why did this get removed

Reply:
Maybe it's just not used in the code

Reply:
When you do an auto code cleanup in VS, it removes unnecessary using directives

Reply:
What @Gurge44 said.